src/visitors/global_var_visitor.hpp

```diff
@@ -0,0 +1,70 @@
+/*************************************************************************
+ * Copyright (C) 2018-2019 Blue Brain Project
+ *
+ * This file is part of NMODL distributed under the terms of the GNU
+ * Lesser General Public License. See top-level LICENSE file for details.
+ *************************************************************************/
+
+#pragma once
+
+/**
+ * \file
+ * \brief \copybrief nmodl::visitor::GlobalToRangeVisitor
+ */
+
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "ast/ast.hpp"
+#include "visitors/ast_visitor.hpp"
+#include "visitors/visitor_utils.hpp"
+
+namespace nmodl {
+namespace visitor {
+
+/**
+ * @addtogroup visitor_classes
+ * @{
+ */
+
+/**
+ * \class GlobalToRangeVisitor
+ * \brief Visitor for Units blocks of AST
+ *
+ * We override AstVisitor::visit_global_var to visit the
+ * ast::GlobalVar nodes of the ast
+ */
+
+class GlobalToRangeVisitor: public AstVisitor {
+  private:
+    /// ast::Ast* node
+    std::shared_ptr<ast::Program> ast;
+
+  public:
+    /// \name Ctor & dtor
+    /// \{
+
+    /// Default UnitsVisitor constructor
+    GlobalToRangeVisitor() = default;
```

Comment:
do we need this? Or do you wanted to delete this?

Reply:
Actually this is not needed, so it can be removed